pkg/policy/visibility.go

```diff
@@ -83,6 +83,7 @@ func NewVisibilityPolicy(anno string) (*VisibilityPolicy, error) {
 		if u8Prot == u8proto.ANY {
 			protos = append(protos, u8proto.TCP)
 			protos = append(protos, u8proto.UDP)
+			protos = append(protos, u8proto.SCTP)
```

Comment:
Did you test L7 policies with this pull request? I thought we had other dependencies on the L4 protocol.

Reply:
Ok. We should list L7 as a limitation then.

Reply:
Sorry, I'm not sure I understand. If L7 policies are being tested as part of `cilium connectivity test` and other tests that pass with this patch, is it still a limitation?

Reply:
But we don't support policies for L7 protocols over SCTP, do we? It just doesn't break existing L7 policies.

Reply:
Ahh. Yes. I see what you mean. Will note that.